ftp/Dockerfile

```diff
@@ -6,4 +6,4 @@ RUN useradd -d /var/ftp -s /sbin/nologin ftp
 COPY ./prep-cert.sh /project/prep-cert.sh
 RUN chmod +x /project/prep-cert.sh
 
-CMD /project/prep-cert.sh && /run.sh pure-ftpd -4 -A -c 50 -C 5 -e -G -H -i -p $FTP_MIN_PORT:$FTP_MAX_PORT -P $PUBLICHOST -r -R -s -X -x -Z $FLAGS
+CMD /project/prep-cert.sh && /run.sh pure-ftpd -4 -A -c 50 -C 5 -e -G -H -i -p $FTP_MIN_PORT:$FTP_MAX_PORT -P 129.16.69.158 -r -R -s -X -x -Z $FLAGS
```

Comment:
Do you mean if we change `129.16.69.158` to something like `$PUBLICFTPHOST`, it should be changed in the Vue as well? I feel like this `PUBLICFTPHOST` only needs to be configured in the `.env`.

Reply:
The previous value for `$PUBLICHOST` was `metabolicatlas.org`, which was incompatible with the FTP server defined at `ftp.metabolicatlas.org`, even though it is the same IP. So, I suspect this variable would need to be the exact same in both the _Documentation_ page (Vue) and in the `pure-ftpd` command. Sharing variables like this is definitely possible but not immediately straightforward - I would suggest to capture it in an independent issue. Or perhaps somebody else would want to implement it in this branch?

Reply:
If we don't want to override `$PUBLICHOST` we could (and probably should) just introduce a new variable in `.env` to represent this IP address. This is just so we can handle potential IP changes and support multiple environments. I don't think this would introduce any incompatibility.

Reply:
I agree and this is exactly what I mentioned in https://github.com/MetabolicAtlas/MetabolicAtlas/pull/596#discussion_r639446693 to use a new variable for instance `$PUBLICFTPHOST`.